src/main/resources/org/zaproxy/zap/extension/hud/resources/Messages.properties

```diff
@@ -37,6 +37,7 @@ hud.optionspanel.button.baseDirectory	= Change
 hud.optionspanel.label.baseDirectory	= Base Directory:
 hud.optionspanel.label.enabledForDesktop = Enable when using the ZAP Desktop
 hud.optionspanel.label.enabledForDaemon = Enable when using ZAP in daemon mode
+hud.optionspanel.label.enableTelemetry = Enable anonymized telemetry
```

Comment:
"anonymized" implies that it was identifiable and was rendered unidentified before submission, which if we take another small step implies that something could go wrong.... Would it be better to say "anonymous"? (Which to me implies that there's no identifying info to start with.)

Reply:
Oops, lost track of this :/
Now changed :)